napalm_logs/config/ios/CONFIGURED_FROM.yml

```diff
@@ -0,0 +1,15 @@
+# <189>24: test-ztp: May 23 15:49:32.302: %SYS-5-CONFIG_I: Configured from tftp://10.83.21.224//cisco-bootstrap by console
+# <189>30: test-ztp: May 23 13:56:15.055: %SYS-5-CONFIG_I: Configured from console by admin on vty0 (10.31.0.24)
+messages:
+  - error: CONFIGURED_FROM
```

Comment:
I don't think we should add another message type for this. I feel that [`CONFIGURATION_COMMIT_COMPLETED`](https://napalm-logs.com/en/latest/messages/CONFIGURATION_COMMIT_COMPLETED.html) might be a better place, and eventually have two new keys, e.g.,

```json
"yang_message": {
      "system": {
          "operations": {
              "commit_complete": true,
              "user": "admin",
              "configured_from": "vty0 (10.31.0.24)",
          }
      }
  }
```

Reply:
Cisco IOS don't have the commit functionality. Besides that the IOS sends this message every time when the user exits the configuration mode even if the configuration wasn't changed.  Therefore I'm not sure if this should go to the `CONFIGURATION_COMMIT_COMPLETED` or if this would be rather confusing with IOS.

Reply:
I see - I assumed that happens on configuration changes (which technically speaking are an implicit commit, executed on every change). Either way, what about having the counterpart of [`USER_ENTER_CONFIG_MODE `](https://napalm-logs.com/en/latest/messages/USER_ENTER_CONFIG_MODE.html), e.g., `USER_EXIT_CONFIG_MODE`?